awspub/image.py

```diff
@@ -358,14 +341,16 @@ def _sns_publish(self) -> None:
         """
         Publish SNS notifiations about newly available images to subscribers
         """
+
+        # Checking if the image(s) are registered or published before sending the notification
         for region in self.image_regions:
```

Comment:
is this whole check still required at all?

Reply:
It depends. If we want to ensure that an image is registered/published before sending a notification, I would keep this as a check for image availability since sns is part of the publication process.

But, if we want to support sns regardless of image availability, we can remove this section.

Reply:
removed this section from our discussion